controller-manager/Makefile

```diff
@@ -3,6 +3,9 @@ include ../Makefile.go.mk
 
 build_deps: build/operatorImageMap.yaml
 
+run:
```

Comment:
I think you should either add a source file (maybe both ./build/operatorImageMap.yaml ./build/controller-manager) or add the target to `.PHONY`.